test/comparison-tests/projectReferences/expectedOutput-3.6/bundle.js

```diff
@@ -106,7 +106,7 @@ eval("\nexports.__esModule = true;\nvar lib_1 = __webpack_require__(/*! ./lib */
 /***/ (function(module, exports, __webpack_require__) {
 
 "use strict";
-eval("\nexports.__esModule = true;\nexports.lib = {\n    one: 1,\n    two: 2,\n    three: 3\n    // I am adding this comment here by hand to ensure\n    // Webpack is using the JS output for project references\n};\n\n\n//# sourceURL=webpack:///./lib/index.ts?");
+eval("\r\nexports.__esModule = true;\r\nexports.lib = {\r\n    one: 1,\r\n    two: 2,\r\n    three: 3\r\n};\r\n\n\n//# sourceURL=webpack:///./lib/index.ts?");
```

Comment:
windows line line feeds ... "\r\n" 
will that cause an issue on CI?

Reply:
It shouldn't, no we normalize line endings in the test runner when we do the comparison; see: https://github.com/TypeStrong/ts-loader/blob/bfa48c75adc424a33bbc1f0ec3eeddb8ea1c2722/test/comparison-tests/create-and-execute-test.js#L435